astropy/io/fits/tests/test_util.py

```diff
@@ -26,3 +27,20 @@ def test():
                     'KeyboardInterrupt ignored until test is complete!')
 
         pytest.raises(KeyboardInterrupt, test)
+
+
+def test_deprecated():
```

Comment:
If the decorator moves to `astropy.utils.decorators`, this test also has to move with it.